test/JDBC/jdbc_schedule

```diff
@@ -8,7 +8,10 @@
 #    new line
 # 6. If you want the framework to not run certain files, use: ignore#!#<test name>
 
-all
+# all
```

Comment:
Would you change back to run all test cases?

Reply:
I have changed it back